src/praisonai-bot/tests/unit/cli/test_gateway_config.py

```
@@ -121,6 +121,103 @@ def test_schema_accepts_gateway_and_hooks_blocks():
+    base = dict(
+        agents={"assistant": {"name": "assistant", "instructions": "Help"}},
+        channels={"telegram": {"token": "fake-token"}},
+    )
+
+    # Misspelled server knob ("timout") -> rejected, names the offending key.
+    with pytest.raises(Exception) as excinfo:
+        GatewayConfigSchema(gateway={"drain_timout": 30}, **base)
+    assert "drain_timout" in str(excinfo.value)
+
+    # Wrong type (string where a float is expected) -> rejected.
+    with pytest.raises(Exception):
```

Ruff (0.15.21)

[warning] Do not assert blind exception: `Exception`

(B017)